### PR TITLE
chore: Dependency update

### LINKED ISSUE #537: chore: Version update
There are a large number of updates from depend-bot. 

Merge them into one and update dependencies.

### PR DESCRIPTION
**Pull request by jrconlin**
## Description

* mass close depend-bot PRs 
* includes vendored googleapis-raw

## Testing

chore

## Issue(s)

Closes #537.

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -0,0 +1,3 @@
+[submodule "vendor/mozilla-rust-sdk/googleapis-raw/grpc"]
+	path = vendor/mozilla-rust-sdk/googleapis-raw/grpc
+	url = https://github.com/grpc/grpc.git
```
Comment (tublitzed) [inline .gitmodules]: it [looks like](https://stackoverflow.com/questions/49490272/how-to-specify-a-tag-in-gitmodules-file) you may be able to specify a tag here. Would it be possible to do this here so we know which specific version of the grpc lib we're depending on?

Comment (jrconlin) [reply .gitmodules]: Thanks, I'll see if I can dig into that. Part of the problem is also that dependbot will flag "old" versions as well. 

I honestly wonder if part of the problem is also home bound folks "catching up on bugs" and releasing a LOT of new revisions of libraries causing all sorts of fun cascades.

Comment (jrconlin) [reply .gitmodules]: no longer an issue with the update of the vendored code.

File: .circleci/config.yml [hunk 2]
```diff
@@ -28,8 +28,9 @@ commands:
       - run:
           name: Core Rust Checks
           command: |
-            cargo audit
             cargo fmt -- --check
+            echo "Skip audit for now since hawk 3.1.0 -> ring 0.16.11 -> abandoned `spin`"
```
Comment (pjenvey) [inline .circleci/config.yml]: The spin +1 other audit on this branch are just warnings, so we can keep audit on

File: Cargo.toml [hunk 3]
```diff
@@ -17,54 +18,56 @@ actix-http = "1"
...
 serde = "1.0"
 serde_derive = "1.0"
 serde_json = { version = "1.0", features = ["arbitrary_precision"] }
 serde_urlencoded = "0.6.1"
 scheduled-thread-pool = "0.2"
-sha2 = "0.8.0"
-slog = { version = "2.5", features = ["max_level_info", "release_max_level_info", "dynamic-keys"] }
-slog-async = "2.3"
+sha2 = "0.8"
+slog = { version = "2.5", features = ["max_level_trace", "release_max_level_error", "dynamic-keys"] }
```
Comment (pjenvey) [inline Cargo.toml]: Bad merge w/ #502

File: .circleci/config.yml (hunk 2 above)
Comment (jrconlin) [reply .circleci/config.yml]: Ok, was getting a fatal off of that, but will retry.

Comment (pjenvey) [reply .circleci/config.yml]: I recall a fatal on this branch at one point but I think it was something else now gone

**Pull request merged**